action-scheduler.php

```diff
@@ -40,7 +40,14 @@ function action_scheduler_register_3_dot_0_dot_1() {
 	}
 
 	function action_scheduler_initialize_3_dot_0_dot_1() {
-		require_once( 'classes/abstracts/ActionScheduler.php' );
+		$autoloader = __DIR__ . '/vendor/autoload.php';
+		if ( is_readable( $autoloader ) ) {
+			require_once( $autoloader );
+			define( 'AS_USING_AUTOLOADING', true );
```

Comment:
Using `spl_autoload_register()` is also using autloading, so this constant could be confusing. What about something like `AS_COMPOSER_AUTOLOADING` instead?

Reply:
> will this be backward compatible? e.g. what happens when someone has AS 2.2

It's compatible with previous versions. Previous versions of AS initialize the autoloader in `ActionScheduler::init( __FILE__ );`. The PR adds the Composer autoloader immediately before that call. 

> could we notify admins (including HEs), likely somewhere in WP Admin, that Composer autoloading is being used (and perhaps the version of AS being loaded)?

We can add these to the help screen & SSR report.

Reply:
> The PR adds the Composer autoloader immediately before that call.

Ah of course. So the newest version of this method still always gets called, in which case, the newest version of the autoloader will always work.

Reply:
@thenbrent I added the AS version to both the help screen and the SSR. I thought it might be confusing to users to see the autoloading info on the help screen so it's only on the SSR.

Reply:
@rrennick yeah you're right, good idea to hide it on the help screen.